src/description.cpp

```diff
@@ -1138,6 +1138,17 @@ void Description::Audio::addPCMUCodec(int payloadType, optional<string> profile)
 	addAudioCodec(payloadType, "PCMU", profile);
 }
 
+void Description::Audio::addAacCodec(int payloadType, optional<string> latmMuxCfg) {
+	if (latmMuxCfg) {
+		string profile = "cpresent=0;config=";
+		profile += *latmMuxCfg;
```

Comment:
If `latmMuxCfg` is just a string concatenated to the profile now, do you really need a different argument for AAC? Couldn't you pass `profile` like other codecs, and set it to `cpresent=1` by default if unset?

Reply:
set a common profile field in rtcTrackInit